doc/getting_started/getting_started.rst

```diff
@@ -3,322 +3,249 @@
-gcc-arm-none-eabi is installed in ``/usr/bin/`` with the common prefix of
-``arm-none-eabi-``.  If your toolchain is at ``/opt/mytoolchain/bin`` with the
-prefix of ``myarch-none-elf-``, it would be
-``CROSS_COMPILE=/opt/mytoolchain/bin/arch-none-elf-``.
-
-Make sure to unset the ZEPHYR_SDK_INSTALL_DIR if you don't use the SDK's host
-tools. See `Building without the Zephyr SDK`_ and `Set Up the Development
-Environment`_ for more details.
-
-Using Custom Cmake Toolchains
-=============================
+The main build products are in :file:`zephyr/samples/hello_world/build/zephyr`.
+The final application binary in ELF format is named :file:`zephyr.elf` by
+default. Other binary formats and byproducts such as disassembly and map files
+will be present depending on the target and build system configuration.
 
-To use a custom toolchain defined in an external cmake file, export the
-following environment variables:
+Other sample projects demonstrating Zephyr's features are located in
+:file:`zephyr/samples` and are documented in :ref:`samples-and-demos`.
 
-   .. code-block:: console
-
-      export ZEPHYR_TOOLCHAIN_VARIANT=<toolchain name>
-      export TOOLCHAIN_ROOT=<path to toolchain>
-
-or set them as cmake variables:
-
-  .. code-block:: console
+Run the Application by Flashing to Another Board
+================================================
 
-      cmake -DZEPHYR_TOOLCHAIN_VARIANT=... -DTOOLCHAIN_ROOT=...
+Most "real hardware" boards supported by Zephyr can be flashed by running
+``ninja flash`` from the build directory. However, this may require
+board-specific tool installation and configuration to work properly.
 
-Zephyr will then include the toolchain cmake file located in:
-``<path to toolchain>/cmake/toolchain/<toolchain name>.cmake``
+See :ref:`application_run` in the Application Development Primer and the
+documentation provided with your board at :ref:`boards` for additional details
+if you get an error.
 
-Running a Sample Application in QEMU
-====================================
+Run the Application in QEMU
+===========================
 
-To perform rapid testing of an application in the development environment you
-can use the QEMU emulation board configuration available for both X86 and ARM
-Cortex-M3 architectures. This can be easily accomplished by calling a special
-target when building an application that invokes QEMU once the build process is
-completed.
+On Linux and macOS, you can run Zephyr applications in emulation on your host
+system using QEMU when targeting both the X86 and ARM Cortex-M3 architectures.
 
-To run an application using the x86 emulation board configuration (qemu_x86),
-type:
+To build run Hello World using the x86 emulation board configuration
+(``qemu_x86``), type:
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
    :host-os: unix
    :board: qemu_x86
    :goals: build run
 
-To exit the qemu emulator, press ``Ctrl-a``, followed by ``x``.
+To exit, type ``Ctrl-a``, then ``x``.
```

Comment:
If you want this to look a bit fancier, you can use:

    To exit, type :kbd:`Ctrl+a`, then :kbd:`x`.